Containers/redis/Dockerfile

```
@@ -1,5 +1,5 @@
 # From https://github.com/docker-library/redis/blob/master/6.2/alpine/Dockerfile
-FROM redis:6.2.8-alpine
+FROM redis:7.0.7-alpine
```

wait for alpine 3.17?

---

Maybe also pin the alpine version?
```suggestion
FROM 7.0.7-alpine3.17
```

---

thought about this as well, however there was a time when minor posgres updates where not pushed with older alpine version anymore. That is why I removed that. Also it is pretty unlikely that alpine updates break this specific container.